### ISSUE TITLE
FEAT: Multiple connections

### ISSUE THREAD

**Issue by matmper** [beekeeper-studio/beekeeper-studio#576]
**Is your feature request related to a problem? Please describe.**
I usually use "Sequel" in MacOS and "Workbench" in Windows and both have multiple connections. I really liked Kepper because is cross plataform, but this feature it's necessary.

**Describe the solution you'd like**
When I click in "new tab" at these softwares, I can make a new database connection, being able to use two or more database at the same time.

**Comment by adrianosmateus** [comment]
I also use Sequel Pro and i felt that.

**Comment by dhrrgn** [comment]
Multiple connections are supported. To open another connection, open a New Window (`ctrl + shift + N` or the via the menu bar)

**Comment by matmper** [comment]
> Multiple connections are supported. To open another connection, open a New Window (`ctrl + shift + N` or the via the menu bar)

Yeah, but its a new window, and ins't pratical. In anothers softwares we can do this in a new tab, and not in a new window.

**Comment by dhrrgn** [comment]
@matmper Ah, then this is a duplicate of #542. 

IMO, because beekeeper opens tables in tabs, mixing table tabs from different connections is just asking for trouble (e.g. running a query on the wrong database). So, if this get added, I would hope that they add a separate set of tabs just for connection. 

Either way, I *really* don't see why tabs are better than separate windows (besides possibly lower memory usage). You can tile windows side-by-side super easily, quickly tab through them, etc. They are *quite* practical. 

As an aside: just because some other apps do things one way, doesn't mean every other app has to do it that way.

**Comment by matmper** [comment]
I understand your point of view, but I believe that this is usage preferences. I prefer a tab connection, if necessary, open a new tab and connect again. The use of multiple windows is not considered practical and without this feature I would not leave Sequel Pro for Beekeeper.

**Comment by rathboma** [comment]
Hey! I think this is possible, and I definitely **will** implement it, but to be totally upfront - it's not a priority right now and probably won't happen until towards the end of the year.

### Notes on building this feature
- Use the global sidebar to namespace database details (tables, views, etc).
-  Use color-coding of the tabs based on the connection color could help to make tab connections obvious.
- Add a drop-down or button on query tabs to change connection
- Default to the currently open connection when creating a query tab

I'll mark as accepted, but don't have time to work on it. If I get a PR for the above workflow I'd be happy to consider it.

**Comment by matmper** [comment]
@rathboma 

Thanks so much, man! With this feature, Beekeeper probably will be my new database station!

**Comment by rathboma** [comment]
I'm going to re-open this :-) Lets keep it open, hopefully someone does a PR for it.

**Comment by snapey** [comment]
First impressions with the software is that this is an accident waiting to happen as @dhrrgn says.  I frequently jump between projects and there is no indication on the tab or the titlebar which database it is connected to (within the same connection).  The sidebar list of tables is in no way connected to the tabs, and a new query window is connected to which database???

**Comment by r3gor** [comment]
DataGrip by JetBrains also has multiple connections with tabs. When working with microservices, this feature is very useful because it allows you to easily connect to and switch between multiple data sources with a simple command. I'm really enjoying Beekeeper, and having this functionality would definitely make it my top choice for managing databases.

**Comment by zscao** [comment]
I accidentally discovered BeeKeeper on my Linux laptop today, and really enjoyed it so far. I was also looking for the tab to connect multiple project. The New Window solution is fine to me, but it would be great if BeeKeeper can have a tab solution similar to MySQL Workbench.

**Comment by MusaGillani** [comment]
👀
